tests/runner.py

```diff
@@ -1029,6 +1029,24 @@ def log_request(code=0, size=0):
 
 def server_func(dir, q, port):
   class TestServerHandler(SimpleHTTPRequestHandler):
+    def send_head(self):
+      if self.path.endswith('.js'):
+        path = self.translate_path(self.path)
+        try:
+          f = open(path, 'rb')
+        except IOError:
+          self.send_error(404, "File not found: " + path)
+          return None
+        self.send_response(200)
```

Comment:
We do send the 200 code later down in do_GET, but I assume this is for the `SimpleHTTPRequestHandler.do_GET(self)` path? Please add a comment.

Reply:
Yeah, that's right - it is the default `do_GET()` handler that this is for - added a comment.